src/Controls/src/Core/Compatibility/Handlers/Shell/Android/SearchHandlerAppearanceTracker.cs

```diff
@@ -83,6 +84,16 @@ protected virtual void SearchHandlerPropertyChanged(object sender, System.Compon
 			}
 		}
 
+		void EditTextFocusChange(object s, AView.FocusChangeEventArgs args)
+		{
+			_searchHandler.SetIsFocused(_editText.IsFocused);
+
+			if (_editText.IsFocused)
```

Comment:
Why do we need to add this behavior? 

The EditText should just open the keyboard itself.

Reply:
Right! I removed it